test/IECoreScene/PointsAlgoTest.py

```diff
@@ -522,5 +522,48 @@ def testRaisesExceptionIfTypesAreIncompatible( self ) :
 		self.assertRaises( RuntimeError, lambda : IECoreScene.PointsAlgo.mergePoints( [pointsA, pointsB] ) )
 
 
+class SegmentPointsTest( unittest.TestCase ) :
+
+	def testCanSegmentUsingIntegerPrimvar( self ) :
```

Comment:
Can we get a test for the non-complete segmentation (e.g. when `data` contains a subset of `primvar.data`) ?

Reply:
added for all data types in  0a7a58b, 5e5f7de & 2a09f43.